### PR TITLE
add a download function

### PR DESCRIPTION
**Pull request by nden**
This is a function which will download the two Roman files from Box.
We should add to the instructions something like

Once the repository is cloned, cd into the repo directory and run

```
from data import download_data

download_data()
```

This should download two files into the data directory:
r0000101001001001001_01101_0001_WFI01_uncal.asdf
r0000101001001001001_01101_0001_WFI01_cal.asdf

### CODE REVIEW

File: data/__init__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import os.path
+import shutil
+import data
+
+data_dir = data.__path__[0]
+
+def download_data():
+    box_path = 'https://data.science.stsci.edu/redirect/Roman/Roman_Data_Workshop/'
+    l1_file = download_file(box_path + "ExampleData/r0000101001001001001_01101_0001_WFI01_uncal.asdf")
```
Comment (zacharyburnett) [inline data/__init__.py:11]: where is `download_file` defined?

Comment (nden) [reply data/__init__.py:11]: oops

Comment (nden) [reply data/__init__.py:11]: added it

**Comment by zacharyburnett** [comment]
perhaps it would be easier if `data/__init__.py` were a single script that could be run with an `if __name__ == '__main__':`, such as `python ./data/download_data.py`

File: data/__init__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    l2_file = download_file(box_path + "ExampleData/r0000101001001001001_01101_0001_WFI01_cal.asdf")
+
+    shutil.copyfile(l1_file, os.path.join(data_dir, "r0000101001001001001_01101_0001_WFI01_uncal.asdf"))
+    shutil.copyfile(l2_file, os.path.join(data_dir, "r0000101001001001001_01101_0001_WFI01_cal.asdf"))
+    print("Done downloading files")
+
```
Comment (zacharyburnett) [inline data/__init__.py:18]: Also, just a suggestion, I have found it is easier to use `Path`

```suggestion
import shutil
import data

from astropy.utils.data import download_file
from pathlib import Path


data_dir = Path(__file__).parent

def download_data():
    box_path = 'https://data.science.stsci.edu/redirect/Roman/Roman_Data_Workshop/'
    l1_file = download_file(box_path + "ExampleData/r0000101001001001001_01101_0001_WFI01_uncal.asdf")

    l2_file = download_file(box_path + "ExampleData/r0000101001001001001_01101_0001_WFI01_cal.asdf")

    shutil.copyfile(l1_file, data_dir / "r0000101001001001001_01101_0001_WFI01_uncal.asdf")
    shutil.copyfile(l2_file, data_dir / "r0000101001001001001_01101_0001_WFI01_cal.asdf")
    print("Done downloading files")

```

Comment (nden) [inline data/__init__.py:18]: @zacharyburnett  Good suggestions. Please submit those changes to a separate PR if necessary

**Review by zacharyburnett — APPROVED** [review]
